main/EventSystem.cpp

```diff
@@ -2442,27 +2440,28 @@ void CEventSystem::ExportDomoticzDataToLua(lua_State *lua_state, uint64_t device
 			_hardwareNames[ID] = tlist;
 		}
 	}
+	
+	Json::Value tempjson;
+	m_webservers.GetJSonDevices(tempjson, "true", "", "dzvID", "", "", "", true, false, false, 0, "");
```

Comment:
Please discuss this on Slack, every developer gets updates when there is a change in a pull request
I would like to have one table that is up2date and can be used for any implementation
Yes, in memory, in a table/structure/eventsystem class member